app/models/user.rb

```diff
@@ -48,8 +49,10 @@ def self.filter_status
 
   validates :username, :email, presence: true
   validates :current_status, inclusion: { in: VALID_STATUS_LIST }
+  validates :avatar,  blob: { content_type: %w[image/png image/jpg image/jpeg], size_range: 1..5.megabytes}
```

Comment:
Does that mean that it is not possible to upload an image smaller than 1 megabyte?

Reply:
That could be a problem right now

Reply:
No, it is possible. It is possible up to 5mb